launcher/launcher_test.go

```diff
@@ -69,6 +69,8 @@ const (
 	statesDir         = "states"
 )
 
+var defaultVars = []string{"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin", "TERM=xterm"}
```

Comment:
It is used in one test only, move it in place:
```go
envVars = []string{"PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin", "TERM=xterm"}
envVars = append...
```

Reply:
it uses in two tests:

TestOverrideEnvVars

TestRuntimeSpec